src/main/java/com/hierynomus/smbj/share/DiskShare.java

```diff
@@ -58,6 +60,25 @@
 
     private static final Logger logger = LoggerFactory.getLogger(DiskShare.class);
 
+    protected ShareInfo shareInfo = null;
+    protected long attrExpiration = 0L;
+    protected static final long attrExpirationPeriod;
+    
+    protected static final long DEFAULT_ATTR_EXPIRATION_PERIOD = 5000;
+    
+    static {
```

Comment:
Removed code that manages attribute expiration. Applications will have to keep track.